walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -223,6 +290,61 @@ public void justEnoughRoom() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void OnlyOptionalAttendeesConsidered() {
```

Comment:
nit: start your method with lowercase o

Also, update the comment below to "We should see three options..." based on the diagram.